tests/arc-validate.Tests/arc-validate.Tests.fsproj

```diff
@@ -16,16 +14,20 @@
     <Compile Include="CLITests/CLITests.fs" />
     <Compile Include="Main.fs" />
   </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Expecto" Version="9.*" />
     <PackageReference Include="YoloDev.Expecto.TestSdk" Version="0.*" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.*" />
     <PackageReference Update="FSharp.Core" Version="8.*" />
     <PackageReference Include="Fake.Core.Process" Version="6.0.0" />
+    <PackageReference Include="FSharpAux" Version="[2.0.0]" />
```

Comment:
I think we do not need these references now that your new tests moved to `ArcExpect.Tests`